core/src/main/resources/io/questdb/site/conf/server.conf

```diff
@@ -711,6 +714,13 @@ cairo.wal.enabled.default=true
 # mmap sliding page size that WalWriter uses to append data for each column
 #cairo.wal.writer.data.append.page.size=1M
 
+# mmap sliding page size that WalWriter uses to append events for each column
+# this page size has performance impact on large number of small transactions, larger
+# this page size has performance impact on large number of small transactions, larger
```

Comment:
This line seems to be duplicate

Reply:
fixed in https://github.com/questdb/questdb/pull/4221/commits/69dca528c919748ad5feffcdecb619b97036c581